lib/app-infos-cache.js

```diff
@@ -6,7 +6,7 @@ import B from 'bluebird';
 
 /** @type {LRUCache<string, import('@appium/types').StringRecord>} */
 const MANIFEST_CACHE = new LRUCache({
-  max: 40,
+  max: 100, // Increased from 40 to accommodate more app manifests
```

Comment:
is there a real need to do that? 
Each manifest belongs to a unique app under test. It is hard to imagine 40 unique apps being tested at the same time. Also, by increasing the cache size we increase the amount of disk space used by driver's temporary files.
A better idea might be to make this value configurable via server arguments or environment variables.